src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -458,7 +459,7 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
   navToServiceConfiguration(service) {
     let serviceInfo = {
       name: service.name,
-      isEnabled: service.status === 'running' ? true : false,
+      isEnabled: ["shutdown", "disabled", "installed", ""].includes(service.status) ? false : true,
```

Comment:
What does empty mean?

Reply:
empty status check is not needed here, I've removed it now.